resources/package_tests/tests/test_package_is_correct.py

```diff
@@ -1,3 +1,4 @@
+import pytest_check as check
 from testinfra.modules.package import RpmPackage
 
 from ..base import *
```

Comment:
Why did you not use absolute imports?

Reply:
Because `package_tests` is not a python module and I'm not sure it should be since we're copying the folder into different location for each test VM/container instance